packages/xo-server-perf-alert/src/index.js

```diff
@@ -470,7 +471,15 @@ ${monitorBodies.join('\n')}`
         return Promise.all(
           map(
             definition.smartMode
-              ? map(this._xo.getObjects({ filter: { type: definition.objectType } }), obj => obj.uuid)
+              ? map(
+                  this._xo.getObjects({
+                    filter: {
+                      type: objectType,
+                      power_state: objectType === 'VM' || objectType === 'host' ? 'Running' : undefined,
```

Comment:
I don't think this works correctly: `getObjects` is passing `filter` to `lodash/iteratee` which will look for objects with the `power_state` property explicitly set to `undefined`.